skops/io/_general.py

```diff
@@ -136,3 +127,17 @@ def type_get_state(obj, dst):
 def type_get_instance(obj, src):
     loaded = _import_obj(obj["content"]["__module__"], obj["content"]["__class__"])
     return loaded
+
+
+GET_STATE_DISPATCH_FUNCTIONS = [
```

Comment:
Could you please add a comment everywhere what the tuples are? I never find iterable of tuples intuitive.

Reply:
Done, please take another look.